helm/qontract-reconcile/templates/template.yaml

```diff
@@ -591,6 +607,8 @@ parameters:
   value: "0"
 - name: USER_ID
   value: dummy
+- name: KUBE_SA_TOKEN_PATH
+  value: '/var/run/secrets/kubernetes.io/serviceaccount'
```

Comment:
So just to clarify, this value should always be the same?

Reply:
Correct. This is the current path utilized by default projection